### PR TITLE
feat(CMS): supported apps by kemnkes

### PR DESCRIPTION
**Pull request by mazipan**
## Description

Add new collection for Application Supported by Kemenkes

Draft on #711

### CODE REVIEW

File: public/admin/config.yml [hunk 1]
```diff
@@ -93,6 +93,23 @@ collections:
             required: false
             hint: Heading banner on top of Home page
             tagname: ""
+      - name: official_supported_apps
+        label: Official Supported Apps by Kemenkes
+        file: _content/official_supported_apps.json
```
Comment (zainfathoni) [inline public/admin/config.yml:98]:
```suggestion
      - name: officially_supported_apps
        label: Officially Supported Apps by Kemenkes
        file: _content/officially_supported_apps.json
```

Just a nitpick on its grammatical error. 😅 🙏

**Review by zainfathoni — CHANGES_REQUESTED** [review]
Thanks, Mas @mazipan!
However, I have a minor nitpick here. 😁

File: public/admin/config.yml (hunk 1 above)
Comment (mazipan) [reply public/admin/config.yml]: Done

**Review by zainfathoni — APPROVED** [review]
LGTM 💯 

Thanks for working on it, Mas @mazipan! 🙏

**Pull request merged**